BackofficeBundle/Resources/public/coffee/FullPageFormView.coffee

```diff
@@ -36,6 +36,7 @@ FullPageFormView = OrchestraView.extend(
     $("form", @$el).ajaxSubmit
       context: button: $(".submit_form",event.target).parent()
       success: (response) ->
+        displayMenu() if $(event.target).attr('name') is 'content_type'
```

Comment:
I would prefer to have a dedicated view for content_type which will be loaded by appConfigurator. This view could override FullPageFormView by adding displayMenu() in initialize.